### ISSUE TITLE
I'm sorry, but this is badly written code

### ISSUE THREAD

**Issue by MrPresidentBanana** [AceLewis/my_first_calculator.py#111]
what you should do is something along the lines of

if sign == '+':
   result = num1+num2
   print(num1 + "+" + num2 + " = " + result)
if sign == '-':
   #...

That way you don't need thousands of lines of code, and it can also process numbers larger than just 50

**Comment by untuned** [comment]
if you didn't guess, this repo is a joke program

**Comment by simondebbarma** [comment]
A joke? On GitHub? 


The audacity!!!

**Comment by debesyla** [comment]
Who are you to question the work of a professional?

**Comment by SixBeeps** [comment]
The code you provided doesn't work on my machine. All it says is "Segmentation fault".

**Comment by AnduRomanYT** [comment]
> what you should do is something along the lines of
> 
> if sign == '+':
> result = num1+num2
> print(num1 + "+" + num2 + " = " + result)
> if sign == '-':
> #...
> 
> That way you don't need thousands of lines of code, and it can also process numbers larger than just 50

I think this meme code rather than an actual shot at programing

**Comment by zbigniewcebula** [comment]
> 
> 
> what you should do is something along the lines of
> 
> if sign == '+':
> result = num1+num2
> print(num1 + "+" + num2 + " = " + result)
> if sign == '-':
> #...
> 
> That way you don't need thousands of lines of code, and it can also process numbers larger than just 50

[https://i.pinimg.com/originals/39/27/f0/3927f0279ed109e35afa4ed19036d29b.png](The You)

**Comment by zbigniewcebula** [comment]
> 
> 
> what you should do is something along the lines of
> 
> if sign == '+':
> result = num1+num2
> print(num1 + "+" + num2 + " = " + result)
> if sign == '-':
> #...
> 
> That way you don't need thousands of lines of code, and it can also process numbers larger than just 50

You:
https://i.pinimg.com/originals/39/27/f0/3927f0279ed109e35afa4ed19036d29b.png

**Comment by bwalter** [comment]
I fully disagree. The proposed code might work but it is unreadable and only the smart person who wrote it can understand it. The original code is much more simple and should be kept. I suggest to close this issue as "won't fix".

**Comment by ghost** [comment]
Try my Calculator, on my repo Shahibur46/Calculator, I pretty sure you all will not be disappointed, please do check, I need some helpers and contributors.

**Comment by Visual-mov** [comment]
What are you talking about? This is the most efficient possible way to write a calculator!

**Comment by H3xad3cimalDev** [comment]
A joke? the audacity
